pkg/apis/servicecatalog/checksum/unversioned/checksum.go

```diff
@@ -38,6 +39,10 @@ func InstanceSpecChecksum(spec servicecatalog.InstanceSpec) string {
 	if spec.Parameters != nil {
 		specString += fmt.Sprintf("parameters:\n\n%v\n\n", string(spec.Parameters.Raw))
 	}
+	if spec.AlphaSecretParameters != nil {
+		// TODO: calc checksum of secret contents instead?
```

Comment:
> if any of these values change due to user update, there's a bug in the controller because we do not implement update

Exactly. Thus I would prefer the secret contents checksum to be addressed along with secret update support in a separate PR (once we find the brave contributor willing to do this). TODO should be sufficient for now to remind and embarass us :)